flowetl/etl/etl/etl_utils.py

```diff
@@ -22,6 +22,7 @@
 from airflow import DAG
 from airflow.hooks.postgres_hook import PostgresHook
 from airflow.operators.python_operator import BranchPythonOperator, PythonOperator
+from airflow.operators.postgres_operator import PostgresOperator
```

Comment:
This import of `PostgresOperator` can now be removed since we're not using it any more (and similarly for `BranchPythonOperator` once we've removed the associated isinstance check below).